test-cases/longevity/longevity-cdc-100gb-8h-multi-dc-topology-changes.yaml

```diff
@@ -0,0 +1,27 @@
+test_duration: 500
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75",
+              "cassandra-stress user no-warmup profile=/tmp/cdc_profile_multidc_preimage_postimage.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=480m -port jmx=6868 -mode cql3 native -rate threads=75"
+             ]
+
+n_db_nodes: '4 4'
+instance_type_db: 'i3.4xlarge'
+n_loaders: '2 2'
+region_aware_loader: true
+n_monitor_nodes: 1
+
+nemesis_class_name: 'TopologyChangesMonkey'
```

Comment:
This should be SisyphusMonkey as I wrote above.

Reply:
fixed